source/standards/pre-commit-hooks.html.md.erb

```diff
@@ -0,0 +1,14 @@
+---
+title: Git Pre-commit Hooks
+last_reviewed_on: 2020-11-27
+review_in: 6 months
+---
+# <%= current_page.data.title %>
+
+[Git hooks](https://githooks.com/) are scripts that Git executes before or after events such as commit, push, and receive. Pre-commit hooks are useful in secrets management because they check for errors and problems before developers commit work locally on their machines.
+
+The Cyber Security team are working on a [pre-commit hook](https://github.com/alphagov/gds-pre-commit) for GDS to use. The main reason for using pre-commit hooks is to prevent secrets being pushed to GitHub repositories.
```

Comment:
It is more correct to say that we have a reccomended tool. See the language of https://github.com/alphagov/gds-pre-commit/blob/master/README.md

Our advice has changed recently, we USED to build a pre-commit hook that offered turnkey support, however the maintenance and support burden was too high, we've now tweaked our repo to provide guidance on how to use off-the-shelf tools.

Reply:
I have changed it according to your recommendations! Could you have another look please? 

Happy new year!